src/main/java/org/b612foundation/adam/datamodel/SinglePropagation.java

```diff
@@ -0,0 +1,76 @@
+package org.b612foundation.adam.datamodel;
+
+import java.util.Objects;
+
+public class SinglePropagation extends AdamObject {
+
+  /** Parameters used to generate the ephemeris of this propagation. Null if unknown. */
+  private PropagationParameters propagationParameters;
+
+  /** Human-readable description of this propagation. */
+  private String description;
+
+  /**
+   * Ephemeris produced as part of the propagation. Formatted as contents of STK .e file. May be null if not requested
+   * that ephemeris be kept.
+   */
+  private String ephemeris;
+
+  /** Calculated state vector at end time as specified in propagation parameters. Format: time, x, y, z, dx, dy, dz. */
```

Comment:
If commenting about the format, might as well include units and something about the time standard/format.

Reply:
Added.